doc/mgmt/Management Framework.md

```diff
@@ -2378,54 +2469,68 @@ mechanism for the app specific configuration.
 5. Translib infra populates the YGOT structure with the payload of the request and performs a syntactic validation
 6. Translib acquires the write lock (mutex lock) to avoid another write happening from the same process at the same time.
 7. Translib infra gets the App module corresponding to the incoming URI.
-8. Translib infra calls the initialize function of the App module with the YGOT structures, path and payload.
-9. App module caches the incoming data into the app structure.
-10. App module calls Transformer function to translate the request from cached YGOT structures into Redis ABNF format. It also gets all the keys that will be affected as part of this request.
-11. App module returns the list of keys that it wants to keep a watch on along with the status.
-12. Translib infra invokes the start transaction request exposed by the DB access layer.
-13. DB access layer performs a WATCH of all the keys in the Redis DB. If any of these keys are modified externally then the EXEC call in step 26 will fail.
-14. Status being returned from Redis.
-15. Status being returned from DB access layer.
-16. Translib then invokes the processWrite API on the App module.
-17. App modules perform writes of the translated data to the DB access layer.
-18. DB access layer validates the writes using CVL and then caches them.
-19. Status being returned from DB access layer.
-20. Status being returned from App module.
-21. Translib infra invokes the commit transaction on the DB access layer.
-22. DB access layer first invokes MULTI request on the Redis DB indicating there are multiple writes coming in, so commit everything together. All writes succeed or nothing succeeds.
-23. Status returned from Redis.
-24. Pipeline of all the cached writes are executed from the DB access layer.
-25. Status retuned from Redis.
-26. EXEC call is made to the Redis DB. Here if the call fails, it indicates that one of the keys that we watched has changed and none of the writes will go into the Redis DB.
-27. Status returned from Redis DB.
-28. Status returned from DB access layer.
-29. Write lock acquired in Step 6 is released.
-30. Status returned from the Translib infra.
-31. REST Status returned from the Request handler.
-32. REST response is sent by the REST gateway to the REST client.
+8. Translib infra invoke App module getNamespace API to get namespace associated with the uri
+9. App module invokes transformer service getNamespaceMapping to retrieve namespace based on Yang model
```

Comment:
getNamespaceMapping returns an array of namespaces only mapped at the request URI only? or does it also return the namespaces of all its children?

Reply:
> getNamespaceMapping returns an array of namespaces only mapped at the request URI only? or does it also return the namespaces of all its children?

Namespace will be based on request URI and children shall use the same namespace